packages/entities/entities-plugins/docs/plugin-form.md

```diff
@@ -83,19 +103,67 @@ The base konnect or kongManger config.
 
 If showing the `Edit` type form, the ID of the plugin.
 
+#### `hideScopeSelection`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Manually toggle visibility of plugin scope selection control.
+
+#### `isCredential`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+A boolean indicating whether the form is being used to create a plugin or an auth credential.
+
+#### `isWizardStep`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Will hide the form buttons if you only want to render the form and want to control form submission from the consuming app.
+
+#### `useCustomNamesForPlugin`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Support instance names for plugins. This can be removed when KHCP-5872-custom-names-for-plugins is removed. Enabled by default for KM.
+
 ### Events
 
 #### error
 
 An `@error` event is emitted when form validation fails. The event payload is the response error.
 
+#### fetch-schema:error
```

Comment:
suggestion: since this is a type of error, I think the syntax should be reversed?
```suggestion
#### error:fetch-schema
```